components/steplist/stories/template.js

```diff
@@ -107,3 +109,43 @@ export const Template = ({
 		</div>
 	`;
 };
+
+export const SteplistGroup = (args, context) => html`
```

Comment:
`SteplistGroup` should be coming from the `steplist.test.js` now so that it is runs the testing grid and be assigned to the Default story.  Other stories that are for docs-only should use the Template directly. `avatar.stories.js` is a good example of this.